GitOut/Features/Git/Stage/GitStageViewModel.cs

```diff
@@ -899,6 +895,24 @@ private bool SetProperty<T>(ref T prop, T value, [CallerMemberName] string? prop
             return false;
         }
 
+        private async Task DeleteFileSnackAsync(string path)
+        {
+            cancelRefreshSnack?.Cancel();
+            cancelRefreshSnack = new CancellationTokenSource();
+            ISnackBuilder? builder = Snack.Builder();
+            builder.AddAction("YES");
+            builder.AddAction("NO");
+            builder.WithMessage("This file is untracked. This will delete the file. Are you sure that you want to delete the file?");
```

Comment:
I would like the file name to be displayed as well, as a "double confirm" for the user. Don't know if we want to scare them with some capital letters as well. Thinking about what vscode did with their "revert untracked files" message 🤔 
```cs
builder.WithMessage($"The file {Path.GetFileName(path)} is untracked. This will delete the file, THERE IS NO UNDO FOR THIS ACTION! Are you sure that you want to delete the file?");
```